### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Ruiseki**

### CODE REVIEW

File: client/src/index.html
Comment (KillianVendewinkele) [inline client/src/index.html:1]: Nice shot

File: client/src/mainscript.js
Comment (KillianVendewinkele) [inline client/src/mainscript.js:1]: Should not be a problem

**Review by progresso85 — APPROVED** [review]
Approve

File: client/src/module/apiRequest.js
Comment (KillianVendewinkele) [inline client/src/module/apiRequest.js:1]: getProductType is the only thing added? no problem

File: client/src/module/createShop.js
Comment (KillianVendewinkele) [inline client/src/module/createShop.js:1]: UwU

**Review by mathias-smj — APPROVED** [review]

File: server/main.js
Comment (KillianVendewinkele) [inline server/main.js:1]: git commit -m "It's ok, you can merge"

**Review by KillianVendewinkele — APPROVED** [review]
EULA > KEQING

**Pull request merged**